brian_cheung/the_basics.md

```diff
@@ -169,8 +169,7 @@ end
 **Answer 12:**
 ```ruby
 def Caps(string)
-	puts "#{string}".upcase
+	puts string.upcase
 end
-
```

Comment:
Looks good! The only nitpick is that you shouldn't capitalize the name of the method here.